apps/marsha/templates/app/secret_cloudfront_private_key.yml.j2

```diff
@@ -8,6 +8,6 @@ metadata:
   name: "{{ marsha_cloudfront_private_key_secret_name }}"
   namespace: "{{ project_name }}"
 data:
-  ssh-privatekey: "{{ DJANGO_CLOUDFRONT_PRIVATE_KEY | default('') | b64encode }}"
+  ssh-privatekey: "{{ MARSHA_VAULT.DJANGO_CLOUDFRONT_PRIVATE_KEY | default('') | b64encode }}"
```

Comment:
in this file you missed the first line, the condition is on the same variable

Reply:
Good catch. Thanks!